doc/Train a Deep Potential model to fit tensor like Dipole and Polarizability.md

```diff
@@ -0,0 +1,109 @@
+# Train a Deep Potential model to fit `tensor` like `Dipole` and `Polarizability`
+
+Unlike `energy` which is a scalar, one may want to fit some high dimensional physical quantity, like `dipole` (vector) and `polarizability` (matrix, shorted as `polar`). Deep Potential has provided different API to allow this. In this example we will show you how to train a model to fit them for a water system. A complete training input script of the examples can be found in 
+
+```bash
+$deepmd_source_dir/examples/water_tensor/dipole/dipole_input.json
+$deepmd_source_dir/examples/water_tensor/polar/polar_input.json
+```
+
+Relevant data are also provided in our examples. But note that **our examples are of very limited amount, and should not be used to train a productive model.**
+
+
+
+The directory of this examples:
+
+-   [The training input script](## the-training-input-script)
+-   [Training Data Preparation](## training-data-preparation)
+- 	[Train the Model](## train-the-model)
+
+## The training input script
+
+Similar to the `input.json` used in `ener` mode, training json is also divided into `model`, `learning_rate`, `loss` and `training`. Most `args` remains the same as `ener` mode, and their meaning can be found [here](https://github.com/deepmodeling/deepmd-kit/blob/devel/doc/train-se-e2-a.md). To fit a tensor, one need to modify `model.fitting_net` and `loss`.
+
+### Model
+
+The `fitting_net` section tells DP which fitting net to use, and should be provided like
+
+```json
+	"fitting_net" : {
+	    "type": "dipole" / "polar",

```

Comment:
This is not a valid json syntax. 
I would suggest to give two pieces of examples one for `dipole` and one for `polar`